src/jsbq.js

```diff
@@ -0,0 +1,52 @@
+const jsbq = module.exports = {}
+
+const jsonSchemaBigquery = require('./index')
+const gbq = require('./gbq')
+const util = require('./util')
+const logger = require('log-driver').logger
+
+jsbq.process = async (project, datasetName, jsonSchema) => {
+  logger.info('Processing json schema...')
+  const tableOptions = jsonSchemaBigquery.convert(jsonSchema)
+  logger.info(JSON.stringify(tableOptions))
+  
+  logger.info('Extracting table name from schema...')
+  const tableName = util.get_table_id(jsonSchema.id)
+  logger.info('Table name is ' + tableName)
+
+  logger.info('Setting table options...')
+  tableOptions.friendly_name = jsonSchema.title
+  tableOptions.description = jsonSchema.description || jsonSchema.title
+  tableOptions.timePartitioning = {
```

Comment:
We need to mention this behaviour in the docs as other users will need to be aware of it.